### PR TITLE
Clarify compatibility explanations

### PR DESCRIPTION
**Pull request by TheMeaningfulEngineer**
Small improvement suggestions which make the compatibility page more unambiguous.

### CODE REVIEW

**Review by tranchitella — APPROVED** [review]

**Comment by TheMeaningfulEngineer** [comment]
Don't know the flow for the docs PR (testing pipelines and prerequisites for merging).
@kacf Let's pls shortly sync on this once you're back.

File: 02.Overview/14.Compatibility/docs.md [hunk 1]
```diff
@@ -128,6 +135,11 @@ The [Mender Artifact format](../03.Artifact/docs.md) is managed by the [Mender A
 
 The compatibility between the Mender server and client is managed by the Device API versions exposed by the server and used by the client. If the Mender server supports the API version of the Mender client, they are compatible.  However, please ensure that the client and server support the [Artifact format](#mender-clientserver-and-artifact-format) version you are using. Device API docs are available in the [API chapter](../../200.Server-side-API/?target=_blank#device-apis).
 
+The higher version API is a superset of the previous one. For endpoints which haven't changed in the new version the previous version ones are assumed.
```
Comment (kacf) [inline 02.Overview/14.Compatibility/docs.md:138]: This is not true, the two APIs are actually incompatible, which is why we bumped the version. But I get your point, it feels a bit like a hybrid because we have kept the v1 endpoints instead of bumping those also to v2.

However, to avoid confusion I would rephrase this a bit: 

> The higher version API contains a mix of old and new API endpoints. For endpoints which haven't changed in the new version the previous version ones are assumed.

Does it seem confusing? Yeah, it is. :slightly_smiling_face: I have created [this ticket](https://tracker.mender.io/browse/MEN-4927) to try to convert it to a more consistent style, where everything is moved to v2.

**Review by kacf — APPROVED** [review]
Pre-approving, but please take a look at the commect below.

**Comment by kacf** [comment]
> Don't know the flow for the docs PR (testing pipelines and prerequisites for merging).
> @kacf Let's pls shortly sync on this once you're back.

If it has run the tests and has approval, you can go ahead and merge. The tests are supposed to run automatically, but occasionally there is a hiccup causing them not to run. Looks like this happened for you. However, if you fix the issue I mentioned above, and re-push, probably they will run this time.

File: 02.Overview/14.Compatibility/docs.md (hunk 1 above)
Comment (TheMeaningfulEngineer) [reply 02.Overview/14.Compatibility/docs.md:138]: Great catch, definitely not a superset and much more correct phrasing.
Changing.
